source/Calamari/Integration/Scripting/WindowsPowerShell/Bootstrap.ps1

```diff
@@ -12,46 +12,137 @@ $ErrorActionPreference = 'Stop'
 # Functions
 # -----------------------------------------------------------------
 
-function Write-VersionTable
+function Log-VersionTable
 {
 	Write-Verbose ($PSVersionTable | Out-String)
 }
 
+function Log-EnvironmentInformation
+{
+	if ($OctopusParameters.ContainsKey("Octopus.Action.Script.SuppressEnvironmentLogging")) {
+		if ($OctopusParameters["Octopus.Action.Script.SuppressEnvironmentLogging"] -eq "True") {
+			return;
+		}
+	}
+
+	Write-Host "##octopus[stdout-verbose]"
```

Comment:
@michaelnoonan Yes, our `Write-Verbose` function simply wraps a call to `Write-Host` with `##octopus[stdout-verbose]` ... 

```
function Write-Verbose([string]$message)
{
	Write-Host "##octopus[stdout-verbose]"
	Write-Host $message
	Write-Host "##octopus[stdout-default]"
}
```
So this ensures they all end up being verbose logs without additional overhead (eg. If you make each of these a `Write-Verbose`, you end up with a lot of additional calls to `Write-Host` for no reason).

@matt-richardson - If you can recommend an alternative we should be using, let us know. This seems to be how we're doing it in all PowerShell scripts in Octopus too (see HealthCheck.ps1).